rest_api/sawtooth_rest_api/rest_api.py

```diff
@@ -29,10 +29,10 @@ def parse_args(args):
                         default=8080)
     parser.add_argument('--host',
                         help='The host for the api to run on',
-                        default="0.0.0.0")
+                        default="127.0.0.1")
     parser.add_argument('--stream-url',
                         help='The url to connect to a running Validator',
-                        default='tcp://localhost:40000')
+                        default='tcp://127.0.0.1:40000')
```

Comment:
Hmm, I was trying to standardize on 127.0.0.1, rather than use localhost in some places and 127.0.0.1 in others. It isn't clear to me why localhost is clearer.